rdkit/Chem/Subshape/SubshapeAligner.py

```diff
@@ -282,42 +274,44 @@ def PruneMatchesUsingShape(self, targetMol, target, queryMol, query, builder, al
   def GetSubshapeAlignments(self, targetMol, target, queryMol, query, builder, tgtConf=-1,
                             queryConf=-1, pruneStats=None):
     import time
+
     if pruneStats is None:
       pruneStats = {}
+
     logger.info("Generating triangle matches")
-    t1 = time.time()
+    t1 = time.perf_counter()
```

Comment:
This is a general comment, but time.time is not a reliable measurement for profiling.  We may think about adjust this to perf_counter in other places, especially unit tests.